src/list-data/entity-type-data.ts

```diff
@@ -280,7 +283,7 @@ export const EntityTypesBcData: EntityI[] = [
 
 export const EntityTypesXproData: EntityI[] = [
   {
-    text: 'Limited Company',
+    text: 'Extraprovincial Limited Company',
```

Comment:
See screenshot attached to comment above for the new labels (to match UI design).